About.html

```diff
@@ -0,0 +1,93 @@
+        <li><a href="#" class="fc-white fs-small no-style">Logout</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">Mypage</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">Korean</a></li>
+      </ul>
+    </div>
+    <div class="nav-items d-flex align-center">
+      <button type="button" id="Menu-button">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </button>
+      <div class="nav-brand">
+        <a href="index.html"><img src="Images/logo.jpg" class="hideOnMobile-f logo"></img></a>
+      </div>
+      <div class="nav-links d-right" id="nav-options">
+        <ul class="nav-ul Mobile-Menu gap-wide display-none" id="mobile-Menu">
+          <li class='d-right'><a href="#" class="hideOnDesktop-I no-style fc-dark" id="close-menu">&times;</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="index.html">Home</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="index.html#Program">Program</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="index.html#Guest-Container">Join</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="index.html#Sponsor">Sponsor</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#">News</a></li>
+          <li><a class="fc-orange fs-medium padded-border no-style" href="#">CC Campaign</a></li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="About-Headline">
+    <p class="Headline-title">Hello! Car Enthusiasts</p>
+    <h1 class="h1-mobile About-Headline-title">Miami Vintage Car Show</h1>
+    <article>
+      Welcome to the Miami Vintage Car Show, where automotive 
+      nostalgia comes to life. Step into a world of classic 
+      beauty and timeless elegance as we showcase an exceptional 
+      collection of vintage cars from different eras. Immerse yourself 
+      in the rich history and artistry of these remarkable automobiles, 
+      and join us for a memorable experience that celebrates the enduring 
+      allure of vintage cars in the heart of Miami.
+    </article>
+    <h3 class="h3 fc-white">Contact Us:</h3>
+    <a href="#" class="fc-white">MiamiVintage@gmail.com</a>
+  </section>
+
+  <section class="CC-summit">
+    <h1 class="fc-dark">Miami Vintage Car Show</h1>
+    <span class="title-underline"></span>
+    <p>The logo of Retro Car Club was decided through the logo competition from
+      8 june to 7 july
+    </p>
+    <img src="Images/logo.jpg" alt="Retro Cars logo">
+  </section>
+
+  <section class="CC-summit bg-light">
+    <h1 class="fc-dark">Hall Of Fame</h1>
+    <span class="title-underline"></span>
+    <p>Take a Look at the last 2 winner cars of Miami Vintage Car Show</p>
+    <div class="winner-cars">
+      <img src="Images/car-desktop.jpg" alt="Vintage Car">
+      <img src="Images/car-desktop.jpg" alt="Vintage Car">
+    </div>
```

Comment:
- [ ] You should try as much as possible to stick to  [the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). For this case I suggest that you attach the oranged top color, texts and numbers on the images to match the original design ☺️ 
Your design 😎 
![image](https://github.com/hamidazim321/Microverse-Capstone-1/assets/99394374/b19aac08-89c5-4433-a6c2-f7dcaae24d61)

Original design 😎 
![image](https://github.com/hamidazim321/Microverse-Capstone-1/assets/99394374/26f83ec2-9116-4ff2-99c4-de3990426468)